cmd/astrolabe/main.go

```diff
@@ -132,7 +132,7 @@ func ls(c *cli.Context) error {
 	if petm == nil {
 		log.Fatalf("Could not find type named %s", peType)
 	}
-	peIDs, err := petm.GetProtectedEntities(context.TODO())
+	peIDs, err := petm.GetProtectedEntities(context.Background())
```

Comment:
Why not initialize `ctx := context.Background()` and use it all across the method?

Reply:
Eh, it's already done and that doesn't really add any clarity.